sql_generators/mobile_kpi_support_metrics/templates/engagement_clients.view.sql

```diff
@@ -20,34 +20,18 @@ WITH active_users AS (
     is_mobile,
   FROM
     `{{ project_id }}.{{ dataset }}.active_users`
+),
+attribution AS (
+  SELECT
+    client_id,
+    sample_id,
+    {% for attribution_field in product_attribution_fields %}
+    {{ attribution_field }},
+    {% endfor %}
+    paid_vs_organic,
+  FROM
+    `{{ project_id }}.{{ dataset }}.attribution_clients`
 )
-{% if product_attribution_fields %}
-  , attribution AS (
-    SELECT
-      client_id,
-      sample_id,
-      channel AS normalized_channel,
-      {% for attribution_field in product_attribution_fields.values() if not attribution_field.name.endswith("_timestamp") %}
-        {% if app_name == "fenix" and attribution_field.name == "adjust_campaign" %}
-          CASE
-            WHEN adjust_network IN ('Google Organic Search', 'Organic')
```

Comment:
this code is to set the adjust_campaign field to a single value in the case of organic attribution, because originally it stores the search keyword an it had thousands of different values, which wasn't good for the aggregated table.

Reply:
LGTM then, can you approve the PR @bani

Reply:
I can't really approve myself without reviewing and I won't have time to review today, but you can approve it and it has the same effect.

Reply:
Merge